config/data_scrubber.go

```diff
@@ -107,9 +163,9 @@ func (ds *DataScrubber) ScrubCmdline(cmdline []string) []string {
 	}
 
 	if changed {
-		return strings.Split(rawCmdline, " ")
+		return strings.Split(rawCmdline, " "), changed
 	}
-	return cmdline
+	return cmdline, changed
```

Comment:
you can reduce this to a single return now since `changed` is being emitted 

```
scrubbed := nil
if changed {
  scrubbed = strings.Split(rawCmdline, " ")
}
return scrubbed, changed
```

Reply:
I've changed to a single return but I keep returning the original cmdline if anything hasn't been scrubbed to avoid crashing the old tests that use the scrubbing feature without cache